devtools/chain/specs/multi_nodes/chain-spec.toml

```diff
@@ -0,0 +1,48 @@
+[genesis]
+timestamp = 1680249207
+extra_data = []
+base_fee_per_gas = "0x539"
+chain_id = 2022
+transactions = "genesis_transactions.json"
+
+[[accounts]]
+address = "0xa0ee7a142d267c1f36714e4a8f75612f20a79720"
+balance = "04ee2d6d415b85acef8100000000"
```

Comment:
Shall we reduce the init balance to resole the explorer issue:
- https://github.com/Magickbase/blockscan/issues/53

cc @KaoImin @sunchengzhu

Reply:
The configuration here sets the init balance for each account at 100 trillion, which is an exceedingly large number. To put it into perspective, the total issuance of CKB is only 36.6 billion. I believe the explorer shouldn't have to support our unreasonable account balances.

Reply:
> But users still can use any balance as they like.
> If it could, it will be.

Yes, this issue is up to the app chain maintainer.

I am just suggesting a small default value for an Axon devnet.
As for the non-devnet case, I think the chain maintainer needs to have full consideration on how to set up the init accounts with how many native tokens.

Reply:
Axon can not limit the size of the initial balance, even if a user wants to set as `U256::MAX - 1`. To reduce the balance in the default config file is out of question.